packages/analytics/src/integrations/amplitude/Amplitude.ts

```diff
@@ -0,0 +1,42 @@
+import { Identify, Group, Page, Track } from 'types';
+import { IIntegration } from '../types';
+import { loadAmplitude } from './loadAmplitude';
+import { AmplitudeConfig } from './types';
+
+export class Amplitude implements IIntegration {
+  constructor(config: AmplitudeConfig) {
+    const amplitude = loadAmplitude();
+    amplitude.init(config.apiKey);
+  }
+
+  identify: Identify = (userId) => {
+    (window as any).amplitude.setUserId(userId);
+  };
+
+  group: Group = (groupId, traits) => {
+    (window as any).amplitude.setGroup(groupId, traits);
+  };
+
+  page: Page = (pageName, traits) => {
+    (window as any).amplitude.track({
+      event_type: '[Amplitude] Page Viewed',
+      event_properties: {
+        page_name: pageName,
+        '[Amplitude] Page Domain': window.location.hostname,
+        '[Amplitude] Page Location': window.location.href,
+        '[Amplitude] Page Path': window.location.pathname,
+        '[Amplitude] Page Title': window.document.title,
+        '[Amplitude] Page URL': window.location.href,
```

Comment:
Question, do we need to hardcoded these? I don't know if Amplitude will work as gtag but I believe for google tag, these parameters are automatically tracked by calling the `page`,

Reply:
they automatically track page views and include all ¡this props, this method is just for the case we need to track something manually